backend/src/main/java/com/woowacourse/kkogkkog/domain/Member.java

```diff
@@ -33,4 +33,8 @@ public Member(Long id, String email, String password, String nickname) {
         this.password = password;
         this.nickname = nickname;
     }
+
+    public boolean isNotSamePassword(String password) {
+        return !this.password.equals(password);
```

Comment:
```suggestion
        return !password.equals(this.password);
```
취향차이인데 저는 이런경우 this.password를 인자로 넣는 걸 좋아해요.

Reply:
정의 말처럼 `null`이 아닐 가능성이 높은 것을 기준으로 작성합니다. 현재 `nullable` 처리가 되어 있으므로 지금 구조가 적절하다고 생각합니다.

Reply:
아하 이해했습니다. 저도 앞으로 null이 아닐 가능성이 높은 기준으로 작성하겠습니다.